hedera-node/src/main/java/com/hedera/services/state/submerkle/CustomFee.java

```diff
@@ -309,4 +310,23 @@ public String toString() {
 					.toString();
 		}
 	}
+
+	public static CustomFee mapFromGrpc(final CustomFeesOuterClass.CustomFee customFeeFromRequest) {
```

Comment:
We can use the [`CustomFee.fromGrpc()` factory](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/main/java/com/hedera/services/state/submerkle/CustomFee.java#L104) here instead. (This `mapFromGrpc()` method will not work quite right for fixed fees with ℏ denom, or fractional fees without a maximum value set.)